src/check/parse/IR.zig

```diff
@@ -1394,6 +1392,23 @@ pub const NodeStore = struct {
         header: HeaderIdx,
         statements: []const StatementIdx,
         region: Region,
+
+        pub fn toStr(self: @This(), env: *base.ModuleEnv, ir: *IR, writer: std.io.AnyWriter) anyerror!void {
+            const header = ir.store.getHeader(self.header);
+            try header.toStr(env, ir, writer);
+
+            try writer.print(" (statements ", .{});
```

Comment:
We should think about how we want to handle spaces if we go with something like this. Feels very brittle for ` (statements` to include the leading space.